mesop/runtime/context.py

```diff
@@ -52,9 +52,14 @@ def scroll_into_view(self, key: str) -> None:
       pb.Command(scroll_into_view=pb.ScrollIntoViewCommand(key=key))
     )
 
+  def focus_component(self, key: str) -> None:
+    self._commands.append(
+      pb.Command(focus_component=pb.FocusComponentCommand(key=key))
+
   def set_theme_density(self, density: int) -> None:
     self._commands.append(
       pb.Command(set_theme_density=pb.SetThemeDensity(density=density))
+
```

Comment:
remove stary line?

Reply:
I'm not following the comment? I guess the comment says "outdated" so maybe I fixed it in a follow up commit after I messed up the merge conflict maybe.

Reply:
ah yeah.